_components/utilities/layout-grid.md

```diff
@@ -4,350 +4,504 @@ layout: styleguide
+{{ grid-markers }}
+
+<div class="docs-grid-example">
+{% capture grid-wrapping %}
+<div class="grid-row">
+  <div class="grid-col-8">.grid-col-8</div>
+  <div class="grid-col-3">.grid-col-3</div>
+  <div class="grid-col-5">.grid-col-5</div>
+</div>
+{% endcapture %}
+{{ grid-wrapping }}
+</div>
+
+<div class="usa-accordion-bordered usa-code-sample margin-top-4">
+  <button class="usa-accordion-button" aria-controls="code-wrapping" aria-expanded="true">Code</button>
+  <div id="code-wrapping" class="usa-accordion-content">
+<div markdown="1">
+```html
+{{ grid-wrapping | strip }}
+```
+</div>
+  </div>
+</div>
+
+<h2 id="gutters">Gutters</h2>
+
+<h3>Default gutter</h3>
+<p class="font-sans-6 measure-3"><code class="docs-inline-code">.grid-gap</code> adds a grid-gap between grid-columns in the grid-row, to a value set as <code class="docs-inline-code">$theme-column-gap</code> in settings (16px default).</p>
+{{ grid-markers }}
+
+<div class="docs-grid-example">
+{% capture grid-gutters %}
+<div class="grid-row grid-gap">
+  <div class="grid-col-4">
+    <div>.grid-col-4</div>
+  </div>
+  <div class="grid-col-4">
+    <div>.grid-col-4</div>
+  </div>
+  <div class="grid-col-4">
+    <div>.grid-col-4</div>
+  </div>
+</div>
+{% endcapture %}
+{{ grid-gutters }}
+</div>
+
+<div class="usa-accordion-bordered usa-code-sample margin-top-4">
+  <button class="usa-accordion-button" aria-controls="code-gutters" aria-expanded="true">Code</button>
+  <div id="code-gutters" class="usa-accordion-content">
+<div markdown="1">
+```html
+{{ grid-gutters | strip }}
+```
+</div>
+  </div>
+</div>
+
+<h3>Large gutter</h3>
+<p class="font-sans-6 measure-3"><code class="docs-inline-code">.grid-gap-lg</code> adds a grid-gap between grid-columns in the grid-row, to a value set as <code class="docs-inline-code">$theme-column-gap-large</code> in settings (32px default).</p>
+{{ grid-markers }}
+
+<div class="docs-grid-example">
+{% capture grid-gutters-lg %}
+<div class="grid-row grid-gap-lg">
+  <div class="grid-col-4">
+    <div>.grid-col-4</div>
+  </div>
+  <div class="grid-col-4">
+    <div>.grid-col-4</div>
+  </div>
+  <div class="grid-col-4">
+    <div>.grid-col-4</div>
+  </div>
+</div>
+{% endcapture %}
+{{ grid-gutters-lg }}
+</div>
+
+<div class="usa-accordion-bordered usa-code-sample margin-top-4">
+  <button class="usa-accordion-button" aria-controls="code-gutters-lg" aria-expanded="true">Code</button>
+  <div id="code-gutters-lg" class="usa-accordion-content">
+<div markdown="1">
+```html
+{{ grid-gutters-lg | strip }}
+```
+</div>
+  </div>
+</div>
+
+<div markdown="1" class="font-sans-6 measure-3">
+<h2 id="sass-mixins">Sass mixins</h2>
+When using the source Sass files, you have the option of using Sass variables and mixins to create custom, semantic, and responsive page layouts. Our predefined grid classes use these same variables and mixins to provide a whole suite of ready-to-use classes for fast responsive layouts.
```

Comment:
This is a little confusing to me. Replace with something like "When generating your USWDS CSS from the source Sass files, you have the option of customizing many of the system defaults by changing the values of theme variables."?

Reply:
So,  the idea behind this is you can set the grid width for a `<footer>` w/o using presentational CSS. I didn't get that from your suggested change though. It's not about changing system defaults but applying them to semmantic elements or classes. Does that help clarify it?

Reply:
I think what was confusing me was that both the ideas (_you can modify the defaults with variables_ and _you can use grid mixins in your component css_) blended together into something that didn't make sense to me.

"When generating your CSS from USWDS source files, you have the option of customizing many system defaults by modifying project theme variables. USWDS also provides grid mixins for adding grid functionality to custom semantic component CSS."

Is that better?